src/arrayMethodSort.js

```diff
@@ -4,8 +4,19 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = (a, b) => `${a}` > `${b}`) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          const tempValue = this[i];
+
+          this[i] = this[j];
+          this[j] = tempValue;
+        }
+      }
+    }
+
+    return this;
```

Comment:
Modifying `Array.prototype` directly can lead to unexpected behavior in code that relies on the original methods, especially when naming it `sort2`. It's usually better to add a method to the prototype of a custom constructor or class that extends Array, or just define a standalone function.